modules/yieldlabBidAdapter.js

```diff
@@ -11,6 +11,7 @@ const BID_RESPONSE_TTL_SEC = 300
 const CURRENCY_CODE = 'EUR'
 const OUTSTREAMPLAYER_URL = 'https://ad.adition.com/dynamic.ad?a=o193092&ma_loadEvent=ma-start-event'
 const GVLID = 70
+const DIMENSION_SIGN = 'x'
```

Comment:
We are using the symbol as separator but it describes a multiplication and is normally called multiplication sign, also known as the times sign or the dimension sign. Since we are using it in context of a size I think dimension sign is the most fitting description.